common/config.py

```diff
@@ -145,6 +145,7 @@ def __init__(self):
 
         # Listeners
         self.listeners_valid_system_sec = int(os.getenv("LISTENERS_VALID_SYSTEM_SECONDS", "300"))
+        self.listener_messages_timeout_sec = int(os.getenv("LISTENER_MESSAGES_TIMEOUT_SECS", "2"))
```

Comment:
The value is 10 on other places, let's make the default value same. But for prod usage I'd consider much higher value.

Reply:
yep, this was set for my local development